lib/voyager_helpers/liberator.rb

```diff
@@ -192,6 +192,7 @@ def get_availability(bibs, full=false)
               else
                 item = get_info_for_item(holding_item_ids.first, c, false)
                 availability[bib_id][mfhd_id][:temp_loc] = item[:temp_location] unless item[:temp_location].nil?
+                availability[bib_id][mfhd_id][:course_reserves] = get_courses(holding_item_ids, c).map(&:to_h)
```

Comment:
Even though it's incomplete, I think at this stage we should only trigger the get_courses query on the first holding_item_id and only when the first item has a `:temp_location` property. Otherwise the query gets run for every bib id, which could get expensive in the search results view.